x-pack/plugins/security_solution/public/detections/components/rules/step_about_rule/default_value.ts

```diff
@@ -33,3 +33,19 @@ export const stepAboutDefaultValue: AboutStepRule = {
   threat: threatDefault,
   note: '',
 };
+
+export const stepDefineStepMLRule: DefineStepRule = {
```

Comment:
Are we using this value only in tests? Maybe it's better to move into the test file?